### PR TITLE
chore: update TS + cdn url 

### PR DESCRIPTION
**Pull request by bautistaaa**
## Related Issue

Closes https://github.com/typehero/typehero/issues/1443

### CODE REVIEW

**Comment by bautistaaa** [comment]
wtf is this pnpm issue

**Comment by bautistaaa** [comment]
> wtf is this pnpm issue
> 
> CleanShot 2024-05-15 at 14 31 55@2x

yeah idk ill revert the upgrade for now

File: tooling/scripts/dl-monaco.ts [hunk 1]
```diff
@@ -4,10 +4,10 @@ import path from 'node:path';
 import fetch from 'node-fetch';
 
 /** @link https://typescript.azureedge.net/indexes/releases.json */
-const TYPESCRIPT_VERSION = '5.3.3';
+const TYPESCRIPT_VERSION = '5.4.5';
```
Comment (Hacksore) [inline tooling/scripts/dl-monaco.ts:7]: i see the vercel build failed, could we only do one change at a time?

```suggestion
const TYPESCRIPT_VERSION = '5.3.3';
```

would require you to rerun the script

Comment (bautistaaa) [reply tooling/scripts/dl-monaco.ts:7]: ugh fine ill just make this a url change

**Pull request closed without merging**